OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs

```diff
@@ -389,7 +389,7 @@ bool SelectProductionBuilding()
 
 			var facility = CurrentQueue.MostLikelyProducer().Actor;
 
-			if (facility == null)
+			if (facility == null && facility.OccupiesSpace != null)
```

Comment:
Still crashing on Bomber John. I think you mean ` || facility.OccupiesSpace == null` here?

Reply:
Yeah, sorry again. 🤦‍♂